test/sanity/code-smell/botmeta.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+"""Make the data in BOTMETA.yml is valid"""
+
+import sys
+import yaml
+
+from voluptuous import Any, Required, Schema
+from voluptuous.humanize import humanize_error
+
+from ansible.module_utils.six import string_types
+list_string_types = list(string_types)
+
+
+def main():
```

Comment:
Another possible check: `files` keys should match real path.

Reply:
I guess we'd need to check this matches a regex: https://github.com/ansible/ansibullbot/pull/1023

Is their a case where a file may exist in an older `stable-2.x` branch but not in `devel`, such as when a module is removed/renamed?

Reply:
That could definitely be the case, particularly when we reorganize files. It makes me wonder if we need per-branch config for the bot -- I'm guessing that would be messy to support and maintain though.

Reply:
I could see if anything fails the current checks.

I'd prefer to not over engineer this (branched config) at this stage.

Reply:
I've added path validation and fixed the issues it found, which seem reasonable https://github.com/ansible/ansible/pull/44903/commits/5a899af8466bc91c4791763d9faf9caa952006d0